remoteworkitem/jira_test.go

```diff
@@ -0,0 +1,29 @@
+package remoteworkitem
+
+import (
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+	jira "github.com/andygrunwald/go-jira"
+)
+
+type fakeJiraIssueFetcher struct{}
+
+func (f *fakeJiraIssueFetcher) listIssues(jql string, options *jira.SearchOptions) ([]jira.Issue, *jira.Response, error) {
+	return []jira.Issue{{}}, &jira.Response{}, nil
+}
+
+func (f *fakeJiraIssueFetcher) getIssue(issueID string) (*jira.Issue, *jira.Response, error) {
+	return &jira.Issue{ID: "1"}, &jira.Response{}, nil
+}
+
+func TestJiraFetch(t *testing.T) {
```

Comment:
I think issues.jboss.org for now. We can look into getting an almighty-test project setup on Atlassian Cloud for the long run... (we can also setup an almighty specific project on issues.jboss.org, but let's hold on that until the rename is clear. You can use something from project ARQ for now. http://issues.jboss.org/browse/ARQ)